pkg/agent/client.go

```diff
@@ -174,8 +174,8 @@ func newAgentClient(address, agentID, agentIdentifiers string, cs *ClientSet, op
 	return a, serverCount, nil
 }
 
-// Connect makes the grpc dial to the proxy server. It returns the serverID
-// it connects to.
+// Connect makes the grpc dial to the proxy server. It returns the count
+// of the number of servers this agent should connected to.
```

Comment:
Sorry to nit pick, but:

```suggestion
// of the number of servers this agent should connect to.
```

(Feel free to just make the change locally and repush, it's just as easy as accepting, rebasing etc; if that's what we do in this repo)